src/Classes/ServiceAPI/MyRadio_Scheduler.php

```diff
@@ -131,7 +131,14 @@ public static function getTerms()
             ORDER BY start ASC'
         );
     }
-
+    
+    /**
+     * Returns term item of the id $termid.
+     *
+     * @todo There's currently no caching on this, could be a potential slowdown
```

Comment:
caching? it's a single, simple sql query, fairly sure no caching is required here. Database has bigger problems if this is an issue

Reply:
My dodgy copy pasting from above, bad me...